python/ray/tune/trial.py

```diff
@@ -454,8 +454,9 @@ def logdir(self):
     @logdir.setter
     def logdir(self, logdir):
         relative_logdir = Path(logdir).relative_to(self.local_dir)
-        assert str(relative_logdir).find("..") == -1
-        logger.warning(
+        if str(relative_logdir).find("..") != -1:
```

Comment:
```suggestion
        if ".." in str(relative_logdir):
```
would that work?

Reply:
@Yard1 thanks for the suggestion. I included it into the last commit.